CONTRIBUTING.md

```diff
@@ -117,6 +117,80 @@ You can learn more about [pull requests on GitHub](https://help.github.com/en/gi
 
 You can also contribute to the project without specifically writing content for the site. Many discussions are happening all around the project. Protocol Labs, and by extension, IPFS, is a collaborative company that relies heavily on the community. We need your opinions and input! If you have an idea about a topic or want to weigh in on a discussion, feel free to have your say either on [GitHub](https://github.com/ipfs/ipfs-docs/issues) or on the [IPFS forums](https://discuss.ipfs.tech/).
 
+## Create a new page
+
+Sometimes you will need to make a whole new page for content to live. The process to do this is outlined below.
+
+These steps were performed in Visual Studio Code (VSCode).
+
+The first step to creating a new page is to identify the type of content it will be to determine where it will live in the structure of the docs. It may need to be a whole new section or it could be added to an existing section as a sub-page. An important concept to keep in mind when deciding where to put the new content is the flow of the already existing content. You wouldn't want to put a page or section designed for experienced users between the already existing **Install** and **Basics** sections.
+
+Once you have determined where the content will live, you must add the appropriate lines to the `config.js` file.
```

Comment:
The flow here is confusing me somewhat. LMK if I'm misunderstanding here. 

_Thoughts:_
Line 128 says ```Once you have determined where the content will live, you must add the appropriate lines to the `config.js` file.```. Then, the reader is directed to create a new branch, which makes sense. Then, the reader adds the line for the page to config.js. 

_Suggestion:_
Seems like it would make more sense for line 128 to lead into the process of creating a new branch and the current line 128 content can be moved to the bottom of the "Create a branch" section to lead into the "Adding a single page" section.

Reply:
(Sorry, I realized that this comment highlights the wrong lines)

Reply:
Addressed this in the new version as well.